mesonbuild/modules/sourceset.py

```diff
@@ -130,6 +131,13 @@ def all_sources_method(self, args, kwargs):
         files = self.collect(lambda x: True, True)
         return list(files.sources)
 
+    @noKwargs
+    @noPosargs
+    def all_dependencies_method(self, args, kwargs):
```

Comment:
Please mark the method as only available in 0.52.0 or newer, using `@FeatureNew`.

Reply:
yep, I got some trouble to get it to work, see coming version.